pkg/virt-controller/services/template_test.go

```diff
@@ -4333,6 +4333,61 @@ var _ = Describe("Template", func() {
 			})
 		})
 
+		Context("with pvc in VMI annotations for sidecar", func() {
+			var vmi *v1.VirtualMachineInstance
+			const (
+				claim           = "test-pvc"
+				dirSidecar      = "/debug"
+				shareDirCompute = "/var/run/debug"
+			)
+			BeforeEach(func() {
+				vmi = api.NewMinimalVMI("pvc-sidecar-test")
+				vmi.Annotations = map[string]string{
+					hooks.HookSidecarListAnnotationName: fmt.Sprintf(`[{"image": "test:test", "pvc": {"name": "%s","volumePath": "%s", "sharedComputePath": "%s"}}]`, claim, dirSidecar, shareDirCompute),
+				}
+			})
+			When("PVC exists on the cluster", func() {
```

Comment:
Should there be a test for the time when PVC is specified in the hook, but it doesn't exist on the cluster? Or am I overthinking here? :)

Reply:
when the PVC doesn't exist the virt-launcher pod will stay in pending state. There isn't an error through for this case (see [Vasily's comment](https://github.com/kubevirt/kubevirt/pull/10598#discussion_r1368159693)). IMO, the test is a bit redundant